docker/README.md

```diff
@@ -75,3 +75,19 @@ Enter docker container to check any issues
 ```
 docker exec -it <container_id|container_name> /bin/bash
 ```
+
+How to Collect Docker Container Performance Metrics using OMI/OMI provider in Docker Container
+-------
+
+OMI/OMI provider can work on containers, just make sure you know following notes:
+- OMI/OMI provider can collect containers performance metrics in a normal container, but you cannot collect virtual machine host's performance metrics in container.
```

Comment:
Yes, omi can collect VM perf metrics like CPU, memory and disk, but it cannot collect VM(hosted the containers) tcp connections number in container, they are different tcp numbers for host and container.